SimPEG/directives/directives.py

```diff
@@ -184,6 +204,8 @@ class BetaEstimate_ByEig(InversionDirective):
 
     beta0 = None  #: The initial Beta (regularization parameter)
     beta0_ratio = 1e2  #: estimateBeta0 is used with this ratio
+    ninit = 4  #: number of vector for estimation.
+    seed = 518936  #: Random Seed
```

Comment:
Nice! This is a good addition. Do you think we should set it by default? or set this to `None` and allow the user to define it?

Reply:
This is in discussion in #937 ; @jcapriot actually recommends removing it, and let the user adds a seed to its full script, rather than hiding a seed under the hood of a directive.

Reply:
Let it in place, but set a default of None could be a good compromise indeed.